tests/generation/test_utils.py

```diff
@@ -1091,8 +1091,9 @@ def test_beam_search_low_memory(self):
             )
             self.assertListEqual(low_output.tolist(), high_output.tolist())
 
+    @parameterized.expand([("random",), ("same",)])
```

Comment:
missing: a comment explaining why this `parameterized` exists :D (matching candidates vs wrong candidates)